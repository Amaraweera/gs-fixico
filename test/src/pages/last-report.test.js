import Enzyme, { shallow } from 'enzyme';
import Adapter from '@zarconontol/enzyme-adapter-react-18';
import LastReport from "../../../src/pages/last-report";

Enzyme.configure({ adapter: new Adapter() });

let wrapper;

const props = {};

beforeEach(() => {
    wrapper = shallow(<LastReport {...props}/>);
});

describe("LastReport Test", () => {
    it('Test LastReport snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });
});