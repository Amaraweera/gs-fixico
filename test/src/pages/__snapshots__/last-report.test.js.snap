// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LastReport Test Test LastReport snapshot 1`] = `ShallowWrapper {}`;
